src/lib/rmt/ssl/current_ca_password_dialog.rb

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2018 SUSE LLC.
+#  All Rights Reserved.
+
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of version 2 or 3 of the GNU General
+#  Public License as published by the Free Software Foundation.
+
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.   See the
+#  GNU General Public License for more details.
+
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, contact SUSE LLC.
+
+#  To contact SUSE about this file by physical or electronic mail,
+#  you may find current contact information at www.suse.com
+
+require 'rmt/utils'
+require 'ui/dialog'
+
+module RMT; end
+module RMT::SSL; end
+
+class RMT::SSL::CurrentCaPasswordDialog < UI::Dialog
+  def initialize
+    textdomain 'rmt'
+
+    @cert_generator = RMT::SSL::CertificateGenerator.new
+  end
+
+  def dialog_content
+    VBox(
+      VSpacing(1),
+      Heading(_('Your CA private key is encrypted. Please input password')),
+      VSpacing(1),
+      HBox(
+        HSpacing(2),
+        VBox(
+          MinWidth(15, Password(Id(:ca_password), _('&Password')))
+        ),
+        HSpacing(2)
+      ),
+      VSpacing(1),
+      HBox(
+        PushButton(Id(:cancel), Opt(:key_F9), Yast::Label.CancelButton),
```

Comment:
If you press cancel without the correct password it will execute `openssl` with the input and when there is no input it will run into the fixed bug

Reply:
should be fixed now